File: src/main/java/io/github/alexiscomete/lapinousecond/worlds/buildings/autorisations/ChoicePlaceAutorisation.java
package io.github.alexiscomete.lapinousecond.worlds.buildings.autorisations;

import org.json.JSONArray;

public class ChoicePlaceAutorisation extends AutList {

    public ChoicePlaceAutorisation(JSONArray jsonArray) {
        for (int i = 0; i < jsonArray.length(); i++) {
            this.buildingAutorisations.add(BuildingAutorisations.toAutorisation(jsonArray.getString(i)));
        }
    }
}

File: src/main/java/io/github/alexiscomete/lapinousecond/worlds/buildings/autorisations/ChoicePlaceAutorisation.kt
package io.github.alexiscomete.lapinousecond.worlds.buildings.autorisations

import org.json.JSONArray

class ChoicePlaceAutorisation(jsonArray: JSONArray) : AutorisationList() {
    init {
        for (i in 0 until jsonArray.length()) {
            buildingAutorisations.add(toAutorisation(jsonArray.getString(i)))
        }
    }
}